ap/plotting/plotter.py

```diff
@@ -0,0 +1,181 @@
+# coding: utf-8
+
+"""
+Generalized plotting functions to create plots from hist histograms
+"""
+
+from ap.util import maybe_import
+
+np = maybe_import("numpy")
+hist = maybe_import("hist")
+plt = maybe_import("matplotlib.pyplot")
+mplhep = maybe_import("mplhep")
+
+
+def draw_error_stairs(ax, h, kwargs={}):
+    norm = kwargs.pop("norm", 1)
+    values = h.values() / norm
+    error = np.sqrt(h.variances()) / norm
+    defaults = {
+        "edges": h.axes[0].edges,
+        "baseline": values - error,
+        "values": values + error,
+        "hatch": "///",
+        "facecolor": "none",
+        "linewidth": 0,
+        "color": "black",
+    }
+    defaults.update(kwargs)
+    ax.stairs(**defaults)
+
+
+def draw_from_stack(ax, h, kwargs={}):
+    norm = kwargs.pop("norm", 1)
+
+    # check if norm is a number
+    try:
+        int(norm)
+        h = hist.Stack(*[i / norm for i in h])
+    except:
```

Comment:
To avoid unforseen exceptions from being falsely handled here, could you catch exact exception types?

Reply:
ah okay, I misunderstood the comment. I'll implement that.